platform/logging/appenders/console/ConsoleAppender.ts

```diff
@@ -0,0 +1,14 @@
+import { LogRecord } from '../../LogRecord';
+import { BaseAppender } from '../base/BaseAppender';
+import { ConsoleAppenderConfig } from './ConsoleAppenderConfig';
+
+export class ConsoleAppender extends BaseAppender {
+  constructor(protected readonly config: ConsoleAppenderConfig) {
+    super(config)
+  }
+
+  append(record: LogRecord) {
+    super.append(record);
+    console.log(this.logRecordToFormattedString(record));
```

Comment:
Instead of `logRecordToFormattedString(logRecord)` we could do:

```
logRecordToFormattedString({
  timestamp: ...,
  level: ...,
  context: ...
  message: ...
})
```

And then you have full control in the appender. And we wouldn't have to add so many `protected` methods